libadikted/Makefile

```diff
@@ -1,14 +1,51 @@
 # Project: libadikted
+# Makefile created by KeeperFX/dkfans team
 
-#To create a debug executable, run with the command:
-# make DEBUG
+# To create a debug executable, run with the command:
+# make DEBUG (THIS IS NOT IMPLEMENTED)
 
-#CROSS_COMPILE = i686-w64-mingw32-
+# Instructions:
+# To create adikted.dll, run:
+# make all
+# To create adikited.so, run:
+# make all BUILD_FOR_LINUX=1 (not fully implemented)
+# To clean up the built files (objects, library, and resource file) run:
+# make clean
 
+# Tested and working with:
+# - MSYS2 (MinGW 32-bit) - make
+# - MSYS2 (MinGW 32-bit) - mingw32-make
+# - MinGW (MSYS) - make
 
-CC           = $(CROSS_COMPILE)gcc
-LINK         = $(CROSS_COMPILE)gcc
-SRC          = \
+# Executable files extension on host environment
+ifneq (,$(findstring Windows,$(OS)))
+  CROSS_EXEEXT = .exe
```

Comment:
Is this necessary?

Reply:
It should allow you to build the Windows version on linux

Reply:
Oh so the compiler executable on Linux is named with the .exe extension?

Reply:
Doh, no I just have the If statement the wrong way around!

So it should be: on windows use .exe, on Linux use "blank"

Reply:
The IF statement is the right way around (I had copied it from KeeperFX), it just reads weirdly. So it does behave how I said it should.

Reply:
Then you can lose it, you don't need the `.exe` to invoke an executable on Windows.

Reply:
I think it is in case there is duplicate files, e.g `.bat` `.com`  `.exe` etc
So not needed, but a nice safety net.

Reply:
Good point.